.github/workflows/release.yml

```diff
@@ -394,8 +397,12 @@ jobs:
                   curl -L --retry 10 --silent --show-error https://github.com/rui314/mold/releases/download/v2.30.0/mold-2.30.0-$(uname -m)-linux.tar.gz | tar -C $(realpath mold) --strip-components=1 -xzf -
             - name: Setup Rust toolchain
               run: rustup show
+            - name: Install Protoc
+              uses: arduino/setup-protoc@v3
+            - name: Install nextest
+              uses: taiki-e/install-action@nextest
             - name: Unit tests
-              run: cargo test --release --all
+              run: cargo nextest run
```

Comment:
This can't work, our tests fail if not run with `--release`. Perhaps this is not running all tests? Or it is using release mode by default?